options.js

```diff
@@ -25,34 +49,63 @@ submitButton.onclick = function() {
     return;
   }
 
-  setExtensionColor(color);
-  log("Color changed to " + color);
+  if(color === crunchyrollOrange) {
+    confirmationMessage.innerText = "You can't remove this color!";
+    log("Tried to remove theme color");
+    return;
+  }
+
+  getColorMenu().then( colorOptions => {
+    const isInMenu = colorOptions.includes(color);
+
+    if(!isInMenu) {
+      confirmationMessage.innerText = "This color isn't in the menu";
+      log("Color not in menu");
+      return;
+    }
+
+    colorOptions = colorOptions.filter(function(element) { return element != color; });
+
+    getExtensionColor().then( currentColor => {
+      const isInMenu = colorOptions.includes(currentColor);
+      if(!isInMenu) setExtensionColor(crunchyrollOrange);
+    });
+
+    setColorMenu(colorOptions);
+    document.location.reload();
+
+  });
+}
+
+function setColorMenu(colorMenu) {
+  chrome.storage.sync.set({ colorOptions: colorMenu }, function () {
+    log("Removing " + color + " from the color menu");
+  });
 }
 
 function setExtensionColor(color) {
   chrome.storage.sync.set({ extensionColor: color }, function () {
     log("Setting extension color to " + color);
   });
-  submitButton.style.backgroundColor = color;
+
+  input.value = color;
+  addButton.style.backgroundColor = color;
+  removeButton.style.backgroundColor = color;
 }
 
 input.addEventListener("keyup", function(event) {
   if (event.keyCode === 13) {
     event.preventDefault();
-    submitButton.click();
+    document.getElementById("").click();
```

Comment:
This is not right o_o

Reply:
I did a mistake, it was supposed to create a preview of the new color, it's fixed now